blog-meeting-minutes/2024-02-09-office-hour.md

```diff
@@ -0,0 +1,36 @@
+---
+slug: office-hour-09-02-2024
+title: Office Hour 09.02.2024
+authors: 
+    - tomasz_barwicki
+tags: [meeting-minutes, community]
+---
+
+## office hour meeting minutes
+
+### System team
+
+- Kube Prometheus Stack upgraded to latest release [56.6.2](https://github.com/prometheus-community/helm-charts/releases/tag/kube-prometheus-stack-56.6.2).
+- Committer Election for Tuncay Tunc on Eclipse Tractus-X has started.
+- The Committer Election for Fábio Mota on project Eclipse Tractus-X concluded successfully.
+- Committer volunteers wanted to participate/shadow next Quality Gate process.
+
+### Security team
+
+- New TRG/s from security team was presented requesting for feedback [Security TRG 8.0](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/657).
+- Suggested to contact security team directly for any support required to use, complete Invicti related issues/tasks.
+- Update for static application security testing/source code scanning, ongoing transition from Veracode to CodeQL. Reach out to security team for any assistance.
+- Reminder on available onboarding process to Snyk .
+- There will be separate security office hours meeting, biweekly Thursdays 8:30 - 9:30.
+
+### FOSS
+
+- N/A
+
+### Open planning / community
+
+- Open meetings [PR](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/658).
+
+### Open discussion
+
+- Question related to TRG 1.04 Diagrams as code, if there a need/requirement to convert already existing .png diagrams. It is recommended to use described in the TRG toolset to keep good level of maintainability of the diagrams, not a hard requirement though in case there is lack of source.
```

Comment:
```suggestion
- Question related to [TRG 1.04 Diagrams as code](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-4), if there a need/requirement to convert already existing .png diagrams. It is recommended to use described in the TRG toolset to keep good level of maintainability of the diagrams, not a hard requirement though in case there is lack of source.
```

Reply:
thanks, done! https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/664/commits/245756bcb927f8cc0cd0bd8d22d3b30896bb9120